IDE/IAR/keys/ecc256_pub_key.c

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
Yes, in fact this is the example public key. The key generator will overwrite this file, as well as `ecc256.der`.
If the example key is compiled in with the IAR project, it will generate a compile warning.